src/qibolab/platform.py

```diff
@@ -147,6 +149,20 @@ def create_tii_qw5q_gold(runcard, simulation_duration=None, address=None, cloud=
         qubits[q].drive = channels[f"L3-{10 + q}"]
         qubits[q].flux = channels[f"L4-{q}"]
         channels[f"L4-{q}"].qubit = qubits[q]
+
+    # Platfom topology
+    Q = sympy.symbols([f"q{i}" for i in range(5)])
+    chip = nx.Graph()
+    chip.add_nodes_from(Q)
+    graph_list = [
+        (Q[0], Q[2]),
+        (Q[1], Q[2]),
+        (Q[3], Q[2]),
+        (Q[4], Q[2]),
+    ]
```

Comment:
Read the former comments https://github.com/qiboteam/qibolab/pull/276#discussion_r1129188076